src/js/containers/agency/v2/AgencyContainerV2.jsx

```diff
@@ -70,15 +69,34 @@ const ComingSoon = () => (
     </div>
 );
 
+const propTypes = {
+    params: PropTypes.shape({
+        agencyId: PropTypes.string
+    })
+};
+
 export const AgencyProfileV2 = ({
-    agencyOverview,
-    agencyId,
-    params,
-    clearAgency,
-    setOverview
+    params
 }) => {
+    const dispatch = useDispatch();
     const [activeSection, setActiveSection] = useState('overview');
     const [selectedFy, setSelectedFy] = useState(FiscalYearHelper.defaultFiscalYear());
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        setLoading(true);
+        // request budgetary resources data for this agency
+        const budgetaryResourcesRequest = fetchBudgetaryResources(params.agencyId);
+        budgetaryResourcesRequest.promise
+            .then((res) => {
+                // parse the response using our data model
+                setLoading(false);
+                const budgetaryResources = Object.create(BaseAgencyBudgetaryResources);
+                budgetaryResources.populate(res.data);
+                // store the data model object in Redux
+                dispatch(setBudgetaryResources(budgetaryResources));
+            });
```

Comment:
`.catch()` how are we handling errors for this content?

Reply:
Added error handling for this, as well as, rendering the error.